rector.php

```diff
@@ -4,19 +4,18 @@
 
 use Rector\CodeQuality\Rector\Class_\InlineConstructorDefaultToPropertyRector;
 use Rector\Config\RectorConfig;
+use Rector\Php84\Rector\Param\ExplicitNullableParamTypeRector;
 use Rector\Set\ValueObject\LevelSetList;
+use Rector\ValueObject\PhpVersion;
 
-return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->paths([
+return RectorConfig::configure()
+    ->withPaths([
         __DIR__ . '/src',
         __DIR__ . '/tests',
-    ]);
-
-    // register a single rule
-    $rectorConfig->rule(InlineConstructorDefaultToPropertyRector::class);
-
-    // define sets of rules
-    $rectorConfig->sets([
-        LevelSetList::UP_TO_PHP_80,
-    ]);
-};
+    ])
+    ->withRules([
+        InlineConstructorDefaultToPropertyRector::class,
+        ExplicitNullableParamTypeRector::class,
+    ])
+    ->withPhpSets(php80: true)
+    ->withPhpVersion(PhpVersion::PHP_84);
```

Comment:
Little bit strange and unexpected configuration but this is suggested by @samsonasik in:

https://github.com/rectorphp/rector/discussions/8823#discussioncomment-10600297

Reply:
@samsonasik thx for the feedback and support. Did rollback the config. If the yii core team want to add another tool I let them decide currently there is no php cs tool in place or I'm not aware of it.